akka-stream/src/main/scala/neotypes/akkastreams/AkkaStreams.scala

```diff
@@ -4,11 +4,30 @@ import akka.stream.scaladsl.{Flow, Source}
 
 import scala.concurrent.{ExecutionContext, Future}
 
+/**
+  * neotype Akka Streams
+  */
 trait AkkaStreams {
+
+  /** neotype support for Akka Streams
+    *
+    * @param ec implicit scala.concurrent ExecutionContext
```

Comment:
It would be good to explain why we need an implicit execution context.
Something along the lines of:

_"The execution context that will be passed down to all **Akka Streams** operations"_.

Reply:
I changed the description.